### ISSUE TITLE
Is this project died?

### ISSUE THREAD

**Issue by kolkov** [lukasz-galka/ngx-gallery#293]
I recreated this package as Angular CLI standard library project with demo app and simple tests. I can make pull request or we can simple continue developing this project in my github repo. Thanks a lot!
https://github.com/kolkov/ngx-gallery

**Comment by omikolaj** [comment]
Does your repo support angular 8? Are you thinking of creating an NPM package? I would like to use it if you have it working with Angular 8? It is currently broken in one of my projects, so I ll have to think about alternatives

**Comment by kolkov** [comment]
@omikolaj yes, it is created with Angular 8+ initially.  Yes, I created Angular library package at npmjs because this repo is frozen at this moment. All API saved.
https://www.npmjs.com/package/@kolkov/ngx-gallery

**Comment by omikolaj** [comment]
Time savior!! I super appreciate your work. I just shaved off couple weeks worth of work trying to find another gallery package, write my own or patch this one up. Thanks a lot! Did you make any additions? Or just left it as is and got it working with angular 8+?

**Comment by kolkov** [comment]
@omikolaj Thanks!
At this moment I try to adapt this lib to work with upcoming Angular 9. But I found many bugs with Ivy enabled. Now I am waiting for issues fixed by Angular team.
https://github.com/angular/angular/issues/34128
In addition I want to add some new functionality to the package, including litebox directive and galleriz.

**Comment by omikolaj** [comment]
Okay. If you can. Try to keep posted in here too.

**Comment by kolkov** [comment]
Ok, but the best way, subscribe for changes of my repo.

**Comment by omikolaj** [comment]
I’ll just do that! Thanks again!

**Comment by gustavguez-solcre** [comment]
@lukasz-galka it is true?
